R/report_text.R

```diff
@@ -44,24 +63,29 @@ summary.report_text <- function(object, ...) {
 
 #' @export
 print.report_text <- function(x, ...) {
-  x <- text_fullstop(x)  # Add full stop if missing
+  x <- text_fullstop(as.character(x))  # Add full stop if missing
   cat(x)
 }
 
+
+#' @export
+print.report <- print.report_text
+
+
 # MISCELLANEOUS ------------------------------------------------------------
 
 
 
 #' @export
-report_text.sessionInfo <- function(x, ...) {
+report_text.sessionInfo <- function(x, table=NULL, ...) {
   sys <- report_system(x)
-  pkg <- report_parameters(x)
+  pkg <- report_parameters(x, table=table)
 
   text <- paste0(sys,
                  ", using the packages ",
                  text_concatenate(pkg),
                  ".\n\nReferences\n----------\n",
-                 as.character(cite_packages(x, ...)))
+                 as.character(cite_packages(x, table=table, ...)))
```

Comment:
missing whitespaces around operator

Reply:
Are you doing this one by one? Or do you have a Daniel-bot for such things?

Reply:
was wondering the same thing